observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -153,41 +153,42 @@ include::snippets/technology-preview.adoc[]
 [source,yaml]
 ----
 # ...
-  ports:
-  - name: promexporter # <1>
-    port: 8889
-    protocol: TCP
   config: |
```

Comment:
The pipe character might be causing the colour issue.

Reply:
The pipe is needed because it is a string. It's fine. In a future version, it will not be needed (I will create a PR for that in the future).